### PR TITLE
DINT-29: try to initialize alloy when correct context is set

### PR DESCRIPTION
**Pull request by aniham**

### CODE REVIEW

File: example/index.js [hunk 1]
```diff
@@ -24,4 +24,15 @@ mse.context.setShopper(mockShopper);
 mse.context.setShoppingCart(mockShoppingCart);
 mse.context.setStorefrontInstance(mockStorefront);
 
+/* beacon poc code below*/
+mse.context.setAEP({
+    imsOrgId: "53A16ACB5CC1D3760A495C99@AdobeOrg",
+    datastreamId: "1144fb8d-b234-4c44-85ac-af91ed64c2dd:dev",
```
Comment (evargast) [inline example/index.js:30]: Are these ok to be public? 🤔

File: tests/handlers/checkout/instantPurchase.test.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import { trackStructEvent } from "@snowplow/browser-tracker";
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
```
Comment (evargast) [inline tests/handlers/checkout/instantPurchase.test.ts:3]: Maybe add a `//TODO` next to all these eslint disables?

Comment (aniham) [reply tests/handlers/checkout/instantPurchase.test.ts:3]: happy to :), but what would be the action - i have the mock here so the tests don't freak out over types - and i think as we add actual tests for AEP stuff (during the next 14 event adds) we'll probably mess with that mock anyway

but anyway - todo test suggestions?? :)

**Review by evargast — APPROVED** [review]
I saw you had a commit labeled `lint` 

you should probs download [lint-staged](https://www.npmjs.com/package/lint-staged)

add a rule along the lines of 

```javascript
    "*.{js,jsx,ts,tsx}": [
        "npm run format:fix",
        "npm run lint:fix"
    ]
```

and to your husky hooks in package json add 

```
        "pre-commit": "lint-staged"
```

--- 
No one wants to be linting after putting a PR up 😄

File: example/index.js (hunk 1 above)
Comment (aniham) [reply example/index.js:30]: i think they're fine... 

it's not an authenticated call, just a destination - if you go to dillards.com you can see that info flying from their site through the wire and it looks like this and it's not for a test org :)

**Comment by aniham** [comment]
@evargast that would be a great suggestion if my first step when downloading a repo wasn't `rm -rf .git/hooks`. i'm a terrible person, i know.

**Pull request merged**